website/src/models/user.js

```diff
@@ -280,7 +280,8 @@ var UserSchema = new Schema({
         down: {type: Number, 'default': 0},
         collect: {type: Schema.Types.Mixed, 'default': {}} // {feather:1, ingot:2}
       },
-      completed: String // When quest is done, we move it from key => completed, and it's a one-time flag (for modal) that they unset by clicking "ok" in browser
+      completed: String, // When quest is done, we move it from key => completed, and it's a one-time flag (for modal) that they unset by clicking "ok" in browser
+      invited: {type: Boolean, 'default': false} // Set to true when invite is pending, set to false when quest invite is accepted or rejected
```

Comment:
That's a much better name. `invited` made sense to me, but `RSVPNeeded` is way better. I'll change that right now.